index.go

```diff
@@ -285,7 +281,7 @@ func (idx *faissIndex) SearchWithoutIDs(x []float32, k int64, exclude []int64, p
 		if err != nil {
 			return nil, nil, err
 		}
-		selector = excludeSelector.sel
+		selector = excludeSelector.GetFaissSelector()
```

Comment:
Line doesn't need any changes again, let's use the function when using the `interface` only - so only applicable to `SearchClustersFromIVFIndex`.

Reply:
with this PR, all the other places you've mentioned don't need a change, do since they all use the Selector interface.

Reply:
Ack.